src/addRemoveUpdateClear.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable import/no-cycle */
+import { implementToDoItems, setToDoArr } from './index.js';
+import {
+  onMouseOver, onMouseOut, onDragStart, onDragEnd,
+} from './dragAndDrop.js';
+import checkItem from './checkItem.js';
+import descriptionOnFocus from './inputsOnFocus.js';
+
+const listWrap = document.querySelector('.todo-list-wrap');
+const addNewInput = document.querySelector('.add-new-input');
+// UPDATE ITEM
+export const updateInput = (item) => {
+  const newArr = JSON.parse(localStorage.getItem('myToDos'));
+  const updateItem = newArr.filter((item2) => item2.index === Number(item.dataset.key))[0];
+  const indexOfNewItem = newArr.indexOf(updateItem);
+  newArr[indexOfNewItem].description = item.value;
+  localStorage.setItem('myToDos', JSON.stringify(newArr));
+};
+// REMOVE ITEM
+export const removeItem = (item, arr, container) => {
+  const newArr = JSON.parse(localStorage.getItem('myToDos'));
+  const removedItem = newArr.filter((item2) => item2.index === Number(item.dataset.key))[0];
+  const removedIndex = newArr.indexOf(removedItem);
+  newArr.splice(removedIndex, 1);
+  container.innerHTML = '';
+  localStorage.setItem('myToDos', JSON.stringify(newArr));
+  setToDoArr(newArr);
+  implementToDoItems(newArr);
+  const toDoDescriptionsInputs = [...document.querySelectorAll('.list-item .description-input')];
+  toDoDescriptionsInputs.forEach((item) => item.addEventListener('focusin', (event) => descriptionOnFocus(event, item)));
+  toDoDescriptionsInputs.forEach((item) => item.addEventListener('focusout', (event) => descriptionOnFocus(event, item)));
+  toDoDescriptionsInputs.forEach((item) => item.addEventListener('input', () => updateInput(item)));
+  arr = [...document.querySelectorAll('.list-item')];
+  arr.forEach((item) => {
+    item.addEventListener('mouseover', (event) => onMouseOver(event, item));
+    item.addEventListener('mouseout', (event) => onMouseOut(event, item));
+    item.addEventListener('dragstart', (event) => onDragStart(event, item));
+    item.addEventListener('drop', (event) => onDragEnd(event, item, arr, listWrap));
+  });
+  const removeBtns = document.querySelectorAll('.remove-item');
+  removeBtns.forEach((item) => item.addEventListener('click', () => removeItem(item, arr, container)));
+  const toDoChecksInputs = [...document.querySelectorAll('.list-item .check-item')];
+  toDoChecksInputs.forEach((item) => item.addEventListener('change', () => checkItem(item)));
```

Comment:
- [ ] You can add a loop to check if the item is checked or completed if it's true remove.